docs/config.md

```diff
@@ -58,6 +58,8 @@ Default logs directory paths are:
 - Windows: `%ProgramData%\OpenTelemetry .NET AutoInstrumentation\logs`
 - Linux: `/var/log/opentelemetry/dotnet`
 
+If there is a problem with creation of the default directory, the path of the current user's [temporary folder](https://docs.microsoft.com/en-us/dotnet/api/System.IO.Path.GetTempPath?view=net-6.0) is used.
```

Comment:
```suggestion
If there is a problem with the creation of the default directory,
the path of the current user's [temporary folder](https://docs.microsoft.com/en-us/dotnet/api/System.IO.Path.GetTempPath?view=net-6.0)
is used.
```

Reply:
fixed in https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/456/commits/3bd2baa6343ce26aa9e37e50f7aed7c829d91951